server/app/models/host_node.rb

```diff
@@ -29,6 +29,8 @@ class Error < StandardError
   field :agent_version, type: String
   field :docker_version, type: String
   field :plugins, type: Hash, default: {}
+  field :volume_drivers, type: Array, default: []
```

Comment:
Some comment like `# [ { 'name' => String } ]` describing the array structure would be nice. Even better would be an `embeds_many :host_node_drivers`.